dist/betajs-media-components.js

```diff
@@ -1079,6 +1079,1151 @@ Scoped.assumeVersion('media:version', '~0.0.191');
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Record Video", ["ba-videorecorder-chooser.record-video"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Screen recorder is not supported on this device", ["ba-videorecorder.screen-recorder-is-not-supported"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("back", ["ba-videoplayer-tracks.back"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Select a frame to use as covershot.", ["ba-videorecorder.pick-covershot-frame"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Progress", ["ba-audioplayer-controlbar.audio-progress"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Pause not supported", ["ba-audioplayer-controlbar.pause-audio-disabled"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Call ended", ["ba-video-call.call-ended"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Total length of", ["ba-videoplayer-controlbar.total-time"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Click here to redo.", ["ba-videorecorder-controlbar.rerecord-tooltip"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Please rotate your device to record in portrait mode.", ["ba-videorecorder.orientation-portrait-required"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Enter fullscreen", ["ba-common-settingsmenu.fullscreen-video"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Unmute sound", ["ba-videoplayer-controlbar.volume-unmute"]);
+        return {};
+    });
+
+    Scoped.define("module:Assets.Languages.", ["module:Assets"], function (Assets) {
+        Assets.strings.register("Record", ["ba-videorecorder-controlbar.record"]);
+        return {};
+    });
+
```

Comment:
Hi @3akat, looks like something is being parsed incorrectly, this PR is adding some weird values to `dist/betajs-media-components.js`